image/v3/modifying/drawing.md

```diff
@@ -94,9 +94,9 @@ $image = $manager->read('test.png');
 
 // draw an orange rectangle with a border
 $image->drawRectangle(10, 10, function ($rectangle) {
-    $rectangle->size(180, 125); // width & height of rectangle
-    $rectangle->background('orange'); // background color of rectangle
-    $rectangle->border('white', 2); // border color & size of rectangle
+    $rectangle->setSize(180, 125); // width & height of rectangle
+    $rectangle->setBackgroundColor('orange'); // background color of rectangle
+    $rectangle->setBorder('white', 2); // border color & size of rectangle
```

Comment:
Please revert. These were correct calls of class `Intervention\Image\Geometry\Factories\RectangleFactory::class`.